### ISSUE TITLE
"all" routes not working

### ISSUE THREAD

**Issue by samfundev** [matthiaaas/express-file-routing#24]
When I try to use an all route (using export default), I get:
```TypeError: Cannot read properties of null (reading 'lazyrouter')```

Changing [this line](https://github.com/matthiaaas/express-file-routing/blob/92bea0dca445c37718a800dd8cd0a98ba1d65e1c/src/router.ts#L53) to:
```typescript
app.all.apply(app, [url, ...getHandlers(exports.default)])
```
Seems to fix it, but I'm not sure if that's the best fix.

**Versions:**
- express-file-routing@2.1.2
 - express@4.18.2

**Comment by matthiaaas** [comment]
Thank you @samfundev for the bug report.

Does the default export behavior work for you in normal routes?

**Comment by samfundev** [comment]
If you mean stuff like:
```typescript
export const get = async (req, res) => {
	...
};
```
Yeah, that works.

**Comment by matthiaaas** [comment]
@samfundev Does it work if you use

```ts
export default async (req, res) => ...
```

in normal routes?

**Comment by samfundev** [comment]
Nope. I've been testing using a regular index file, doesn't work.

**Comment by matthiaaas** [comment]
Fixed in v2.1.3. Thank you for the report @samfundev!
